Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessCluster.java

```diff
@@ -340,7 +340,7 @@ public String getName() {
 
     @Override
     public String getTitle() {
-        return name + " [" + state.toUpperCase() + "]";
+        return String.format("%s [%s]", name, getMasterState().toUpperCase());
```

Comment:
How about to use the cluster status, such as `Scheduled` rather than `unknown`, if `master == null`?
@djlan , how about your idea?

Reply:
Synced offline. We will use cluster state when master state is `null`